roles/ci_gen_kustomize_values/templates/hci/edpm-values/values.yaml.j2

```diff
@@ -24,20 +24,28 @@ data:
 {%  endfor %}
 {% endif %}
     nodes:
-{% for compute in cifmw_networking_env_definition.instances.keys() %}
-{%   if compute.startswith('compute') %}
-      edpm-{{ compute }}:
+{% set instances_names = []                                                                             %}
+{% set _original_nodeset = (original_content.data | default({})).nodeset | default({})                  %}
+{% set _original_nodes = _original_nodeset.nodes | default({})                                          %}
+{% set _has_computes = _original_nodes.keys() | select('contains', 'compute') | length > 0                %}
+{% set _has_cephs = _original_nodes.keys() | select('contains', 'ceph') | length > 0                      %}
```

Comment:
IMHO, we could do 
```
{% set _vm_type = _original_nodes.keys() | first | split('-')[1] %}
```

Alternatively, would it be possible to add `networker` type also.